Sming/Arch/Esp8266/Components/gdbstub/gdbstub-internal.h

```diff
@@ -12,6 +12,22 @@
 
 #pragma once
 
+#define XCHAL_CP0_SA_SIZE               0
```

Comment:
When I initially created this PR these were missing from the SDK. I'll re-check and use those if they've been added.

Reply:
No, these are still defined in core.h but oddly only after they're required. So still need them here.